<?xml version="1.0" encoding="utf-8"?>

<launch>

  <!-- Load Parameter -->
  
  <param name="use_cpu"           value="False" />

  <!-- Start yolov5 and ros wrapper -->
  <node pkg="yolov8_ros" type="yolo_depth_detect.py" name="yolov8_ros" output="screen" >
  <!-- <node pkg="yolov8_ros" type="yolo_deepSort.py" name="yolov8_ros" output="screen" > -->
  <!-- <node pkg="yolov8_ros" type="yolo_v8.py" name="yolov8_ros" output="screen" > -->
    <param name="weight_path"       value="$(find yolov8_ros)/weights/yolov8s.pt"/>
    <param name="image_topic"       value="/camera/color/image_raw" />
    <param name="pub_topic"         value="/yolov8/BoundingBoxes" />
    <param name="camera_frame"       value="/camera/color/image_raw"/>
    <param name="conf"              value="0.5" />
    <param name="depth_image_width"     type="double"         value="640" />
    <param name="depth_image_height"     type="double"         value="480" />
  </node>

</launch>
